docs/rules/space-in-parens.md

```diff
@@ -12,7 +12,9 @@ var x = (1 + 2) * 3;
 
 ## Rule Details
 
-This rule will enforce consistency of spacing directly inside of parentheses, by disallowing or requiring one or more spaces to the right of `(` and to the left of `)`. In either case, `()` will still be allowed.
+This rule will enforce consistent spacing directly inside of parentheses, by disallowing or requiring one or more spaces to the right of `(` and to the left of `)`.
+
+As long as you do not explicitly allow or disallow empty parentheses using the `"empty"` exception , `()` will be allowed.
```

Comment:
This could be confusing, sounds like the rule can disallow empty parentheses (not just allow/disallow spaces).

Maybe 'As long as you do not explicitly disallow `()`...'

Reply:
True. I actually would like to simply change this behavior (which would be a breaking change) so that empty is not a special case and works the same way all other exceptions do, i.e. `always` would reject `foo()`, but that is probably best left to another PR and would also warrant some discussion. Updated docs again to be clearer.